src/hps_align/plot/tracks.py

```diff
@@ -0,0 +1,157 @@
+import os
+from ._plotter import Plotter
+from .index_page import htmlWriter
+
+
+@Plotter.user
+def tracks(p: Plotter):
+    """
+    """
+    if not os.path.exists(p.outdir + "/TrackPlots"):
+        os.makedirs(p.outdir + "/TrackPlots")
+
+    plotFolder = "trk_params/"
+    charges = ["", "_neg", "_pos"]
+    vols = ["_top", "_bottom"]
+    variables = ["Chi2",
+                 "nHits",
+                 "phi",
+                 "tanLambda",
+                 #  "trk_extr_bs_x",
+                 #  "trk_extr_bs_y",
+                 #  "trk_extr_bs_x_rk",
+                 #  "trk_extr_bs_y_rk",
+                 #  "d0",
+                 #  "z0",
+                 "p"]
+
+    for crg in charges:
+        for vol in vols:
+            for var in variables:
+                hname = plotFolder + var + vol + crg
+
+                if ("pos" in crg):
+                    corrcrg = "q-"
+                elif ("neg" in crg):
+                    corrcrg = "q+"
+                else:
+                    corrcrg = "All"
+
+                # File loop
+                histos = [f.Get(hname) for f in p.input_files]
+                p.make_1D_plots(histos,
+                                out_name='TrackPlots/'+var+vol+crg,
```

Comment:
`out_name='TrackPlots/'+var+vol+crg` -> `out_name='/TrackPlots/'+var+vol+crg`
missing / in path